### PR TITLE
Fix compatibility with other packages

### PR DESCRIPTION
**Pull request by mtomala**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -9,13 +9,13 @@
   ],
   "require": {
     "php": "^7.1|^8.0",
-    "goetas-webservices/soap-reader": "^0.3.1",
+    "goetas-webservices/soap-reader": "dev-patch-1",
```
[the remark this answers is not in the document]
Comment (mtomala) [reply composer.json]: will you create new tag on goetas-webservices/soap-reader or should I leave 0.3.x-dev here?

File: composer.json [hunk 2]
```diff
@@ -9,13 +9,13 @@
   ],
   "require": {
     "php": "^7.1|^8.0",
-    "goetas-webservices/soap-reader": "^0.3.1",
+    "goetas-webservices/soap-reader": "0.3.x-dev",
```
Comment (goetas) [inline composer.json:12]: I have tagged now `v0.3.3`, you can use that

Comment (goetas) [reply composer.json:12]: see https://github.com/goetas-webservices/soap-reader/releases/tag/v0.3.3

**Comment by goetas** [comment]
sorry for the small comments, i think that after the last change this is ready to go!

File: composer.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (goetas) [reply composer.json]: i have created a new release

**Comment by mtomala** [comment]
> sorry for the small comments, i think that after the last change this is ready to go!

No problem.
I've commited changes requirements

File: composer.json (hunk 2 above)
Comment (mtomala) [reply composer.json]: done

**Comment by goetas** [comment]
thanks!

**Pull request merged**